book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental)
```

Comment:
**OPTIONAL** - Good job done so far, but I suggest that you do not need to push the created rental into the rentals attribute. That was already done when you created a new rental instance in the code directly above. Please fix this 🙏